qe/io.py

```diff
@@ -52,13 +55,23 @@ def load_integrals(
         f = bz2.open(fcidump_path)
     else:
         f = open(fcidump_path)
+        used_zip = False
 
     # Only non-zero integrals are stored in the fci_dump.
     # Hence we use a defaultdict to handle the sparsity
     n_orb = int(next(f).split()[2])
 
-    for _ in range(3):
-        next(f)
+    # Key were looking for
+    key = " /\n"
+    found = False
+    # Keep looping until the key is found
+    while not found:
+        # Read line
+        line = f.readline()
+        # Decode if needed
+        line = line.decode("utf-8") if used_zip else line
+        # Check if it equals the key
+        found = line == key
```

Comment:
You can use the takewhile I think
`https://docs.python.org/3/library/itertools.html#itertools.takewhile`

def if_new_line(line) {
        line = line.decode("utf-8") if used_zip else line
        return line.strip() # Is empty line work as sentinel? if not go back to ` /n`
}
takewhile(if_new_line, f) 
```
Arguably a little more read-able / functional